bin/pfs_integration_test.sh

```diff
@@ -88,33 +88,43 @@ ingestImages.py $TARGET --mode=link \
 # Build bias
 constructBias.py $TARGET --rerun $RERUN/bias \
     --id dateObs=2015-12-22 arm=r spectrograph=1 \
+    --calibId calibDate=2015-12-22 arm=r spectrograph=1 \
```

Comment:
I'm surprised you need `calibDate`. I thought it should be calculated automatically.